pandas/core/strings.py

```diff
@@ -2401,11 +2401,137 @@ def rindex(self, sub, start=0, end=None):
 
     _shared_docs['ismethods'] = ("""
     Check whether all characters in each string in the Series/Index
-    are %(type)s. Equivalent to :meth:`str.%(method)s`.
+    are %(type)s.
+
+    This is equivalent to running the Python string method
+    :meth:`str.%(method)s` for each element of the Series/Index. If a string
+    has zero characters, `False` is returned for that check.
 
     Returns
     -------
-    is : Series/array of boolean values
+    Series
+        Series of boolean values with the same length as the original
+        Series/Index.
+
+    See Also
+    --------
+    Series.str.isalpha : Check whether all characters are alphabetic.
+    Series.str.isnumeric : Check whether all characters are numeric.
+    Series.str.isalnum : Check whether all characters are alphanumeric.
+    Series.str.isdigit : Check whether all characters are digits.
+    Series.str.isdecimal : Check whether all characters are decimal.
+    Series.str.isspace : Check whether all characters are whitespace.
+    Series.str.islower : Check whether all characters are lowercase.
+    Series.str.isupper : Check whether all characters are uppercase.
+    Series.str.istitle : Check whether all characters are titlecase.
+
+    Examples
+    --------
+    **Checks for Alphabetic and Numeric Characters**
+
+    >>> s1 = pd.Series(['AB', 'C12', '42', ''])
+
+    >>> # All are alphabetic characters
```

Comment:
I find these comments before each function unnecessary.

Reply:
Sure, I can strip them out where it's more self-explanatory, though I'm thinking to keep the ones in the section on *More Detailed Checks for Numeric Characters* as I found this the most confusing part. i.e. why there are so many checks for numeric values and what the difference between them is. Or I could put these as text instead of comments.

Reply:
Sounds good to me, the ones that you think the example is not clear by itself I think it's better to have the explanations as text and not as code comments.